apps/core/lib/core/persons/persons_api.ex

```diff
@@ -90,13 +90,11 @@ defmodule Core.Persons.PersonsAPI do
     end
   end
 
-  @doc """
-  used only for graphql
-  """
+  # Used only for graphql
   def search(filter, order_by, cursor) do
     Person
     |> preload(^~w(documents phones addresses merged_persons master_persons)a)
-    |> BaseFilter.filter(filter)
+    |> PersonFilter.filter(filter)
```

Comment:
Since any filter module have only one function which can be called, I propose to import this function instead of aliasing whole module

```suggestion
    |> filter(filter)
```